cognite_toolkit/_cdf_tk/prototypes/commands/import_.py

```diff
@@ -81,17 +82,17 @@ def execute(
             if not overwrite and destination_transformation.exists():
                 self.warn(LowSeverityWarning(f"File already exists at {destination_transformation}. Skipping."))
                 continue
-            destination_transformation.write_text(yaml.safe_dump(transformation))
+            safe_write(destination_transformation, yaml.safe_dump(transformation))
             if source_query_path is not None:
                 destination_query_path = destination_folder / f"{destination_transformation.stem}.sql"
-                destination_query_path.write_text(safe_read(source_query_path))
+                shutil.copyfile(source_query_path, destination_query_path)
```

Comment:
This was causing the failure, as the query file contained none characters not supported by the default windows encoding.